### ISSUE TITLE
Some links fail 1.4.1 Use Of Color

### ISSUE THREAD

**Issue by fstrr** [w3c/wai-wcag-quickref#232]
Some links fail [WCAG 1.4.1. Use Of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) because:

1. They have no non-color way to identify them.
2. There is a contrast ratio of less than 3:1 between the link text and the surrounding non-link text.

Links to Understanding Techniques is an example:
<img width="793" alt="screen shot of a sentence with a link at the end of it" src="https://user-images.githubusercontent.com/20049493/171904523-2ea5d74f-8873-4f39-83b3-cfa916ad82f3.png">

The non-link text is #1d1d1d and the link text is #005a9c, which gives a ratio of 2.36:1. The [Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73) covers this scenario.

**Comment by Dreamleaf** [comment]
![image](https://github.com/w3c/wai-wcag-quickref/assets/104745/01cc5692-8f2f-4fa0-9831-5930385c348d)

I was coming to report the same issue, the section where the links fail is as per the attached image.
